### PR TITLE
fixed disabled dropdown, added validation

### PR DESCRIPTION
**Pull request by SashaBondarchuk**
https://trello.com/c/SlhmNrYn/204-bug78-create-challenge-page-the-language-dropdown-list-is-disabled-on-the-create-challenge-solutions-and-create-challenges-test

https://trello.com/c/LUDsoqDM/235-bug100-solutions-page-the-download-list-with-languages-is-not-active-if-the-user-decided-to-change-the-language

image

### CODE REVIEW

**Review by AyTee001 — APPROVED** [review]

File: backend/LeetWars.Core/LeetWars.Core.DAL/Migrations/20230927152001_DeleteLanguageVersions.cs
Comment (tatianahutii) [inline backend/LeetWars.Core/LeetWars.Core.DAL/Migrations/20230927152001_DeleteLanguageVersions.cs:1]: we cannot change existing migration

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: backend/LeetWars.Core/LeetWars.Core.DAL/Migrations/20230927152001_DeleteLanguageVersions.cs
Comment (SashaBondarchuk) [reply backend/LeetWars.Core/LeetWars.Core.DAL/Migrations/20230927152001_DeleteLanguageVersions.cs:1]: i haven't changed any info, it is only visual studio code cleanup

Comment (SashaBondarchuk) [reply backend/LeetWars.Core/LeetWars.Core.DAL/Migrations/20230927152001_DeleteLanguageVersions.cs:1]: it removed empty spaces

**Comment by SashaBondarchuk** [comment]
[[BUG#100] [Solutions page] The download list with languages ​​is not active if the user decided to change the language](https://trello.com/c/LUDsoqDM/235-bug100-solutions-page-the-download-list-with-languages-is-not-active-if-the-user-decided-to-change-the-language)

**Review by tatianahutii — APPROVED** [review]

**Pull request merged**